packages/gatsby-source-drupal/src/gatsby-node.js

```diff
@@ -97,6 +97,12 @@ exports.sourceNodes = async (
           }
         }
         data = data.concat(d.data.data)
+        // Add support for includes. Includes allow entity data to be expanded
+        // based on relationships.
```

Comment:
I find the naming (`included`) confusing with the comment. This is not your fault (!) but could you add something like "(exposed as `included`)" to this line to make it clear to the reader that the `included` is not a typo.

Reply:
Thanks for the suggestion Peter! I absolutely agree that this might cause confusing. I've added some extra info in the code comments.